### PR TITLE
Update URL in http test to something which exists

### PR DESCRIPTION
**Pull request by mikepurvis**

### CODE REVIEW

**Comment by scpeters** [comment]
enabling pull request testing in https://github.com/ros/rosdistro/pull/12794

File: test/test.cpp [hunk 1]
```diff
@@ -88,7 +88,7 @@ TEST(Retriever, http)
   try
   {
     Retriever r;
-    MemoryResource res = r.get("http://pr.willowgarage.com/downloads/svnmerge.py");
+    MemoryResource res = r.get("http://repositories.ros.org/repos.key");
```
Comment (tfoote) [inline test/test.cpp]: That is not a documented URL and should not be considered stable. I'd recommend `http://packages.ros.org/ros.key` instead.

**Review by tfoote — CHANGES_REQUESTED** [review]

File: test/test.cpp (hunk 1 above)
Comment (mikepurvis) [reply test/test.cpp]: Done

**Review by tfoote — APPROVED** [review]

**Comment by scpeters** [comment]
@tfoote what does it take for a CI job to run?

**Comment by tfoote** [comment]
Your PR landed today, typically devel jobs (including PR builds) are reconfigured overnight. I've triggered the reconfigure manually to get it added sooner: http://build.ros.org:8080/job/Kdev_reconfigure-jobs/ It'll probably be available in the next hour or so. Once it's there, you can do `@ros-pull-request-builder retest this please` as a member of the org.  That's only necessary because this was created before the webhooks existed.

**Comment by tfoote** [comment]
@ros-pull-request-builder retest this please

**Comment by scpeters** [comment]
+1

**Comment by scpeters** [comment]
ok, I'm learning this new github interface

**Review by scpeters — APPROVED** [review]

**Comment by mikepurvis** [comment]
Thanks all.

**Pull request merged**